### DISCUSSION TITLE
Can we apply blending mode just on specific sublayer filename

### DISCUSSION THREAD

**Discussion by luckyadirake** [nftchef/art-engine#164] [Help + Q&A]
I have an art which needs to have cloth's shadow

for example

Cloth
--Cloth A
--- Cloth A
**--- Cloth A Shadow = blending mode to be applied here and layer ordering must be below the actual cloth**

How can i achieve this ?

**Comment by luckyadirake** [comment]
For example

I have a model

Base ( with different skin colors)

Eyes

Eye A
----Eye A
----Eye A Shadow * blending mode to be applied here *

Eye B
----Eye B
----Eye B Shadow * blending mode to be applied here *

Clothes

Cloth A
--- Cloth Design A
--- Cloth Shadow A * blending mode to be applied here *
These shadow need to be blended with Base (skin colors)

**Comment by nftchef** [comment]
Yes actually, This is an undocumented feature still, but
to add a blending mode to a single file, you can append a query string in the file name, staring with `?`

example:
```
EyeBShadow?blend=multiply&opacity=25.png
```
blend mode names are the same as those supported by node canvas https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation

you can use one or both `blend=` , `opacity=`. If you use both, combine them with a `&` like in the example above.
opacity is a whole number/integer (percentage) 0-100
